renovate.json

```diff
@@ -10,6 +10,7 @@
   "python": {
     "enabled": false
   },
+  "ignoreDeps": ["Mime", "Mime-Detective", "Moq"],
```

Comment:
I'd suspect we can get rid of Mime now. #4283 should now bring Mime's native libs into the build correctly, so version pinning isn't a concern anymore.